test/box/error.test.lua

```diff
@@ -275,3 +275,30 @@ box.schema.func.drop('runtimeerror')
 --
 err = box.error.new('TestType', 'Message arg1: %s. Message arg2: %u', '1', 2)
 err.message
+
+-- gh-6632: add copy constructor for error object
+--
```

Comment:
We write all new tests in luatest. Please rewrite in luatest. This test doesn't need `box.cfg` so it should probably live in `test/app-luatest`.

Reply:
Done. See test/app-luatest/gh_6632_add_copy_constructor_for_error_object_test.lua.